server/src/main/java/org/opensearch/cluster/coordination/FollowersChecker.java

```diff
@@ -92,15 +92,16 @@ public class FollowersChecker {
         "cluster.fault_detection.follower_check.interval",
         TimeValue.timeValueMillis(1000),
         TimeValue.timeValueMillis(100),
-        Setting.Property.NodeScope
+        Setting.Property.NodeScope,
+        Setting.Property.Dynamic
     );
 
     // the timeout for each check sent to each node
     public static final Setting<TimeValue> FOLLOWER_CHECK_TIMEOUT_SETTING = Setting.timeSetting(
         "cluster.fault_detection.follower_check.timeout",
         TimeValue.timeValueMillis(10000),
         TimeValue.timeValueMillis(1),
-        TimeValue.timeValueMillis(60000),
+        TimeValue.timeValueMillis(150000),
```

Comment:
we can keep the max at 2mins - 120000

Reply:
we have used 150s in testing, so added this for safety in case we do need to use a high number at some point. can go ahead and change this if you think it's safe to do so.